packages/api-logs/__tests__/utils.test.js

```diff
@@ -0,0 +1,55 @@
+import { getHeaderValue, getFormattedUserAgent } from '../utils';
+
+describe('API Logs Utils', () => {
+  describe('getHeaderValue', () => {
+    let headers;
+
+    beforeEach(() => {
+      headers = [
+        { name: 'Content-Type', value: 'image/jpeg' },
+        { name: 'User-Agent', value: 'Mozilla' },
+      ];
+    });
+
+    it('returns the value of a specified http header', () => {
+      expect(getHeaderValue(headers, 'User-Agent')).toBe('Mozilla');
+    });
+
+    it('normalizes case of header key', () => {
+      expect(getHeaderValue(headers, 'content-type')).toBe('image/jpeg');
+    });
+
+    it('returns null if the header is not present', () => {
+      expect(getHeaderValue(headers, 'not-here')).toBeNull();
+    });
+  });
+
+  describe('getFormattedUserAgent', () => {
+    it('outputs an empty string when fed a falsy value user agent', () => {
+      expect(getFormattedUserAgent('')).toBe('');
+      expect(getFormattedUserAgent(undefined)).toBe('');
+      expect(getFormattedUserAgent(null)).toBe('');
+    });
+
+    it('trims whitespace', () => {
+      expect(getFormattedUserAgent(' ')).toBe('');
+      expect(getFormattedUserAgent(' a ')).toBe('a');
+    });
+
+    it('handles readme api explorer', () => {
+      expect(getFormattedUserAgent('ReadMe-API-Explorer')).toBe('ReadMe API Explorer');
+    });
+
+    it('handles node-fetch', () => {
+      expect(getFormattedUserAgent('node-fetch/x.x.x')).toBe('node');
+    });
+
+    it('handles python-request', () => {
+      expect(getFormattedUserAgent('python-request/x.x.x')).toBe('python');
```

Comment:
I feel like we should capitalize these "pretty-printed" user agent strings if we're going through the trouble to iconize 'em? am I crazy?

Reply:
(@yishinae might have opinions, if she's around!?)

Reply:
I'm indeed around! Although I'm not sure if I fully understand what we're talking about capitalizing!

Reply:
The text next to the green icon in [this screengrab](#issue-565580862). But...........now that I look again it seems this already _is_ capitalized via CSS or something, so I guess this is really a non-issue. Marking this as resolved, but now that I know you're around expect many grammar-related tags!

Reply:
I'll be here!